cmd/libs/go2idl/make-client/output/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
maybe? I think it's better to have it? If nothing else it's where to put
package-godoc, which we do definitely want.

On Fri, Nov 20, 2015 at 4:03 PM, krousey <notifications@github.com> wrote:

> In cmd/libs/go2idl/make-client/output/doc.go
> <https://github.com/kubernetes/kubernetes/pull/17534#discussion_r45534464>
> :
>
> > @@ -0,0 +1,22 @@
> > +/*
>
> @lavalamp <https://github.com/lavalamp> Do we have that rule? It seems
> weird to have a doc.go file when there is only one or mainly one file for
> the package.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17534/files#r45534464>.
>